configs/external_kvstore_with_spif.json

```diff
@@ -33,11 +34,15 @@
                     "- KVStore is on an external SPI Flash block device",
                     "The flash regions are as follows:",
                     "1. Bootloader - 32KiB from the beginning of flash",
-                    "2. KVSTORE - 2 flash sectors immediately following the bootloader",
-                    "3. Active App Metadata Header - (1KiB/2KiB) from the end of KVSTORE",
-                    "4. Active App - From end of header to the end of flash",
+                    "2. Active App Metadata Header - (1KiB/2KiB) from the end of the bootloader",
+                    "3. Active App - From end of header to KVSTORE",
+                    "4. KVSTORE - 2 flash sectors at the end of flash",
                     "+--------------------------+",
                     "|                          |",
+                    "|         KVSTORE          |",
+                    "|                          |",
+                    "+--------------------------+ <-+ storage_tdb_internal.internal_base_address",
```

Comment:
This comment is not correct, as it refers to TDB_INTERNAL configuration, while this file is using FILESYSTEM.